data/monster/humanoids/minotaur_cult_prophet.lua

```diff
@@ -1,6 +1,46 @@
 local mType = Game.createMonsterType("Minotaur Cult Prophet")
 local monster = {}
 
+local combat = Combat()
```

Comment:
You need to extract the spell method from the monster folder, it should be in the "scripts" folder

Reply:
done